api/test/e2enew/base/http.go

```diff
@@ -92,3 +104,44 @@ func BatchTestServerPort(times int) map[string]int {
 
 	return res
 }
+
+func GetReader(reqParams map[string]string, contentType string, files []UploadFile) (io.Reader, string) {
+	if strings.Index(contentType, "json") > -1 {
+		bytesData, _ := json.Marshal(reqParams)
+		return bytes.NewReader(bytesData), contentType
+	}
+	if files != nil {
+		body := &bytes.Buffer{}
+		writer := multipart.NewWriter(body)
+		for _, uploadFile := range files {
+			file, err := os.Open(uploadFile.Filepath)
+			if err != nil {
+				panic(err)
```

Comment:
`panic` here is not very good.
`return err` would be better.

Reply:
done.